Crash when running `npm run test`

I was trying to do a PR for vscode-dotty, but I cannot run the tests.

Note that `/home/jendrik/dev/scala/dotty` is the folder which contains this repo.

```
$ cd vscode-dotty && npm run test

> dotty@0.1.3 test /home/jendrik/dev/scala/dotty/vscode-dotty
> node ./node_modules/vscode/bin/test

### VS Code Extension Test Run ###
Current working directory: /home/jendrik/dev/scala/dotty/vscode-dotty
Running extension tests: /home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/code /home/jendrik/dev/scala/dotty/vscode-dotty/test --extensionDevelopmentPath=/home/jendrik/dev/scala/dotty/vscode-dotty --extensionTestsPath=/home/jendrik/dev/scala/dotty/vscode-dotty/test
[main 23:30:34] update#setState idle

bash: cannot set terminal process group (-1): Inappropriate ioctl for device
bash: no job control in this shell

[main 23:31:04] update#setState checking for updates

Error: Error: Cannot find module '/home/jendrik/dev/scala/dotty/vscode-dotty/test'
	at t.handleExtensionTests (/home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/resources/app/out/vs/workbench/node/extensionHostProcess.js:716:507)
	at /home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/resources/app/out/vs/workbench/node/extensionHostProcess.js:712:854
	at Object.g [as _notify] (/home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/resources/app/out/vs/workbench/node/extensionHostProcess.js:69:202)
	at Object.enter (/home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/resources/app/out/vs/workbench/node/extensionHostProcess.js:72:556)
	at n.Class.derive._oncancel._run (/home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/resources/app/out/vs/workbench/node/extensionHostProcess.js:74:380)
	at n.Class.derive._oncancel._completed (/home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/resources/app/out/vs/workbench/node/extensionHostProcess.js:73:823)
	at c (/home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/resources/app/out/vs/workbench/node/extensionHostProcess.js:77:267)
	at q.then.o.(anonymous function) (/home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/resources/app/out/vs/workbench/node/extensionHostProcess.js:77:481)
	at Object.g [as _notify] (/home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/resources/app/out/vs/workbench/node/extensionHostProcess.js:69:202)
	at Object.enter (/home/jendrik/dev/scala/dotty/vscode-dotty/.vscode-test/VSCode-linux-x64/resources/app/out/vs/workbench/node/extensionHostProcess.js:72:556)

[main 23:31:07] update#setState idle

Tests exited with code: 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dotty@0.1.3 test: `node ./node_modules/vscode/bin/test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dotty@0.1.3 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jendrik/.cache/npm/_logs/2018-03-12T22_31_10_662Z-debug.log
```

--- a/.gitignore
+++ b/.gitignore
@@ -73,3 +73,6 @@ compiler/after-pickling.txt
 *.dotty-ide-version
 
 *.decompiled.out
+
+# The vscode app for testing
+vscode-dotty/.vscode-test

--- a/vscode-dotty/test/extension.test.ts
+++ b/vscode-dotty/test/extension.test.ts
@@ -1,0 +1,21 @@
+//
+// Note: This example test is leveraging the Mocha test framework.
+// Please refer to their documentation on https://mochajs.org/ for help.
+//
+
+// The module 'assert' provides assertion methods from node
+import * as assert from 'assert';
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+import * as vscode from 'vscode';
+import * as dotty from '../src/extension';
+
+// Defines a Mocha test suite to group tests of similar kind together
+suite("vscode-dotty tests", function () {
+
+    // Defines a Mocha unit test
+    test("dummy test", function() {
+        assert.equal(1, 1)
+    });
+});

--- a/vscode-dotty/test/index.ts
+++ b/vscode-dotty/test/index.ts
@@ -1,0 +1,22 @@
+//
+// PLEASE DO NOT MODIFY / DELETE UNLESS YOU KNOW WHAT YOU ARE DOING
+//
+// This file is providing the test runner to use when running extension tests.
+// By default the test runner in use is Mocha based.
+//
+// You can provide your own test runner if you want to override it by exporting
+// a function run(testRoot: string, clb: (error:Error) => void) that the extension
+// host can call to run the tests. The test runner is expected to use console.log
+// to report the results back to the caller. When the tests are finished, return
+// a possible error to the callback or null if none.
+
+import * as testRunner from 'vscode/lib/testrunner';
+
+// You can directly control Mocha options by uncommenting the following lines
+// See https://github.com/mochajs/mocha/wiki/Using-mocha-programmatically#set-options for more info
+testRunner.configure({
+    ui: 'tdd', 		// the TDD UI is being used in extension.test.ts (suite, test, etc.)
+    useColors: true // colored output from test results
+});
+
+module.exports = testRunner;

